android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -184,6 +183,72 @@ fun purchasePackage(
     }
 }
 
+fun purchaseSubscriptionOption(
+    activity: Activity?,
+    productIdentifier: String,
+    optionIdentifier: String,
```

Comment:
so will the hybrids pass through just the identifier, not the subscriptionoption itself?

Reply:
Yup yup! This is how they worked for `purchasePackage` and `purchaseProduct` 🤷‍♂️  I assume its so that each individual just needs to take the strings from their objects and pass them to PHC

Reply:
We are also exposed to users passing a mismatching productId/optionId I guess... But we do log an error, which i think it's the best we can do, considering we are losing the original objects.